pkg/controllers/user/logging/deployer/deployer.go

```diff
@@ -150,7 +151,25 @@ func (d *Deployer) deployRancherLogging(systemProjectID, appCreator string) erro
 		return err
 	}
 
-	app := rancherLoggingApp(appCreator, systemProjectID, templateVersion.ExternalID, driverDir, dockerRootDir)
+	var clusterLogging *mgmtv3.ClusterLogging
+	if cls, err := d.clusterLoggingLister.List("", labels.NewSelector()); err == nil {
+		for _, cl := range cls {
+			if cl.Spec.ClusterName == cluster.Name {
+				clusterLogging = cl
+			}
```

Comment:
* Do we also need to also check projectLogging, not just clusterLogging ? Seems like issue only calls out cluster, but I see that types have it on project also. Was thought that its not bad idea to add to project anyway to make uniform, or was there some other reason ? 
* Instead of doing this loop, would think it will be fast to just pass the object in from sync. That would also solve the project case. Could also pass key and do a `Get` but thats extra call and you'd have to do on both project/cluster lister.

Reply:
Looking at part 2 again, sync is called without the logger key/obj in `NodeSync` and `ClusterSync`, so maybe this is the best way.

Reply:
Project logging just modifies the fluentd configuration, this changes only affects the deployment of containers which is handled by cluster logging.